[2016-09-06 08:41:27,377] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-06 08:49:35,175] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-06 08:49:35,259] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-06 08:51:20,837] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ahn9co0
Internal Server Error (500) for request GET /buyer/task2/6550

Template execution error (In /app/views/TaskCenter/buyerTaskDetail2.html around line 404)
Execution error occured in template /app/views/TaskCenter/buyerTaskDetail2.html. Exception raised was NullPointerException : Cannot get property 'title' on null object.

play.exceptions.TemplateExecutionException: Cannot get property 'title' on null object
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:86)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskCenter.buyerTaskDetail2(TaskCenter.java:1707)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.NullPointerException: Cannot get property 'title' on null object
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:260)
	... 13 more
[2016-09-06 11:22:17,869] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [task.itemUrl] Required
[2016-09-06 11:22:24,323] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [task.itemUrl] Required
[2016-09-06 11:23:35,662] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [task.itemUrl] Required
[2016-09-06 11:26:15,294] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [task.itemPic] Must be checked
[2016-09-06 13:31:31,591] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-06 13:32:57,346] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ahn9co1
Internal Server Error (500) for request GET /buyer/task/perform2/6553

Template compilation error (In /app/views/TaskExecutor/perform2.html around line 2)
The template /app/views/TaskExecutor/perform2.html does not compile : expecting ''', found '\n' 

play.exceptions.TemplateCompilationException: expecting ''', found '\n' 
	at play.templates.GroovyTemplate.compile(GroovyTemplate.java:186)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:224)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskExecutor.perform2(TaskExecutor.java:236)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
[2016-09-06 13:33:33,193] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ahn9co2
Internal Server Error (500) for request GET /buyer/task/perform2/6553

Template compilation error (In /app/views/TaskExecutor/perform2.html around line 2)
The template /app/views/TaskExecutor/perform2.html does not compile : expecting ''', found '\n' 

play.exceptions.TemplateCompilationException: expecting ''', found '\n' 
	at play.templates.GroovyTemplate.compile(GroovyTemplate.java:186)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:224)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskExecutor.perform2(TaskExecutor.java:236)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
[2016-09-06 13:33:37,969] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ahn9co3
Internal Server Error (500) for request GET /buyer/task/perform2/6553

Template compilation error (In /app/views/TaskExecutor/perform2.html around line 2)
The template /app/views/TaskExecutor/perform2.html does not compile : expecting ''', found '\n' 

play.exceptions.TemplateCompilationException: expecting ''', found '\n' 
	at play.templates.GroovyTemplate.compile(GroovyTemplate.java:186)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:224)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskExecutor.perform2(TaskExecutor.java:236)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
[2016-09-06 13:58:08,961] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:00:57,966] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajc0bo5
Internal Server Error (500) for request GET /buyer/task/perform2/6559

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token ")", delete this token

play.exceptions.CompilationException: Syntax error on token ")", delete this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
[2016-09-06 14:01:00,045] ERROR [jobs-thread-7] play.Logger.niceThrowable(Logger.java:604) - 

@71ajc0bo6
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token ")", delete this token

play.exceptions.CompilationException: Syntax error on token ")", delete this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 14:01:00,236] ERROR [jobs-thread-6] play.Logger.niceThrowable(Logger.java:604) - 

@71ajc0bo8
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token ")", delete this token

play.exceptions.CompilationException: Syntax error on token ")", delete this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 14:01:31,219] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajc0boa
Internal Server Error (500) for request GET /buyer/tasks2

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token ")", delete this token

play.exceptions.CompilationException: Syntax error on token ")", delete this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
[2016-09-06 14:02:11,894] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:19:45,001] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:20:23,766] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:21:10,362] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:23:55,015] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:26:17,953] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:26:37,685] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:29:47,877] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:30:47,317] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:31:41,267] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:33:41,037] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:34:13,272] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 14:35:36,222] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajgefe6
Internal Server Error (500) for request GET /buyer/task/perform2/6559

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	... 1 more
[2016-09-06 14:35:36,222] ERROR [play-thread-1] play.Logger.error(Logger.java:457) - Error during the 500 response generation
java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.templates.GroovyTemplate$TClassLoader.defineTemplate(GroovyTemplate.java:79)
	at play.templates.GroovyTemplate.directLoad(GroovyTemplate.java:92)
	at play.templates.BaseTemplate.loadFromCache(BaseTemplate.java:56)
	at play.templates.TemplateLoader.load(TemplateLoader.java:81)
	at play.templates.TemplateLoader.load(TemplateLoader.java:179)
	at play.server.PlayHandler.serve500(PlayHandler.java:792)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:247)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:42:40,676] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:42:40,678] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:579)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:45:05,278] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:45:05,278] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:579)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:47:51,780] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:47:51,780] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:579)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:51:35,573] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:51:35,573] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 579)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:579)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:56:12,260] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:56:12,260] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:58:40,603] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 14:58:40,613] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 14:59:31,286] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@71ajji96n
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Type mismatch: cannot convert from List&lt;String> to boolean

play.exceptions.CompilationException: Type mismatch: cannot convert from List<String> to boolean
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 14:59:31,387] ERROR [jobs-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajji96p
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Type mismatch: cannot convert from List&lt;String> to boolean

play.exceptions.CompilationException: Type mismatch: cannot convert from List<String> to boolean
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:00:26,412] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:00:26,412] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:01:20,637] ERROR [jobs-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajji971
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Type mismatch: cannot convert from List&lt;String> to boolean

play.exceptions.CompilationException: Type mismatch: cannot convert from List<String> to boolean
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:01:20,781] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@71ajji973
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 572)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Type mismatch: cannot convert from List&lt;String> to boolean

play.exceptions.CompilationException: Type mismatch: cannot convert from List<String> to boolean
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:01:46,918] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:01:46,928] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:02:44,483] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:02:44,483] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:03:50,707] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:03:50,707] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:05:23,210] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:05:23,210] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:06:53,927] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:06:53,927] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:07:39,951] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:07:39,951] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:10:08,675] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:10:08,675] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:18:45,713] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:18:45,723] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:20:50,754] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:20:50,755] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
java.lang.NullPointerException: Try to read itemUrls on null object vos.BuyerTaskStepVo (controllers.TaskExecutor.saveStep2, line 572)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:23:58,503] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:23:58,504] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated object is null
java.lang.IllegalArgumentException: The validated object is null
	at org.apache.commons.lang.Validate.notNull(Validate.java:192)
	at org.apache.commons.lang.Validate.notNull(Validate.java:178)
	at models.BuyerTaskStep2.create(BuyerTaskStep2.java:90)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:645)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:28:33,678] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:28:33,679] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated object is null
java.lang.IllegalArgumentException: The validated object is null
	at org.apache.commons.lang.Validate.notNull(Validate.java:192)
	at org.apache.commons.lang.Validate.notNull(Validate.java:178)
	at models.BuyerTaskStep2.create(BuyerTaskStep2.java:90)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:643)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:37:04,381] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-06 15:37:04,381] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated object is null
java.lang.IllegalArgumentException: The validated object is null
	at org.apache.commons.lang.Validate.notNull(Validate.java:192)
	at org.apache.commons.lang.Validate.notNull(Validate.java:178)
	at models.BuyerTaskStep2.create(BuyerTaskStep2.java:90)
	at controllers.TaskExecutor.saveStep2(TaskExecutor.java:642)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-06 15:38:58,322] ERROR [main] play.Logger.error(Logger.java:426) - Could not bind on port 80
[2016-09-06 15:39:03,311] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-06 15:43:58,070] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25b
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 612)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token "}", { expected after this token

play.exceptions.CompilationException: Syntax error on token "}", { expected after this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:43:58,136] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25d
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/TaskExecutor.java around line 612)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Syntax error on token "}", { expected after this token

play.exceptions.CompilationException: Syntax error on token "}", { expected after this token
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:46:33,447] ERROR [main] play.Logger.error(Logger.java:426) - Could not bind on port 80
[2016-09-06 15:54:38,957] ERROR [jobs-thread-9] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25f
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/snippet/Snippet.java around line 4)
The file /app/snippet/Snippet.java could not be compiled. Error raised is : List cannot be resolved to a type

play.exceptions.CompilationException: List cannot be resolved to a type
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:428)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:54:39,188] ERROR [jobs-thread-5] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25h
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/snippet/Snippet.java around line 4)
The file /app/snippet/Snippet.java could not be compiled. Error raised is : List cannot be resolved to a type

play.exceptions.CompilationException: List cannot be resolved to a type
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:428)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 15:59:01,944] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25j
Internal Server Error (500) for request POST /user/upload/token

Oops: ClassNotFoundException
An unexpected error occured caused by exception ClassNotFoundException: snippet.Snippet

play.exceptions.UnexpectedException: Unexpected Error
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:100)
	at play.classloading.ApplicationClassloader.getAnnotatedClasses(ApplicationClassloader.java:491)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:127)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:515)
	at play.Play.start(Play.java:537)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.ClassNotFoundException: snippet.Snippet
	at java.lang.ClassLoader.findClass(ClassLoader.java:358)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at play.classloading.ApplicationClassloader.loadClass(ApplicationClassloader.java:91)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:98)
	... 7 more
[2016-09-06 15:59:06,226] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25k
Internal Server Error (500) for request GET /task/publish2

Oops: ClassNotFoundException
An unexpected error occured caused by exception ClassNotFoundException: snippet.Snippet

play.exceptions.UnexpectedException: Unexpected Error
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:100)
	at play.classloading.ApplicationClassloader.getAnnotatedClasses(ApplicationClassloader.java:491)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:127)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:515)
	at play.Play.start(Play.java:537)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.ClassNotFoundException: snippet.Snippet
	at java.lang.ClassLoader.findClass(ClassLoader.java:358)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at play.classloading.ApplicationClassloader.loadClass(ApplicationClassloader.java:91)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:98)
	... 7 more
[2016-09-06 15:59:23,325] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25l
Internal Server Error (500) for request GET /buyer/tasks2

Oops: ClassNotFoundException
An unexpected error occured caused by exception ClassNotFoundException: snippet.Snippet

play.exceptions.UnexpectedException: Unexpected Error
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:100)
	at play.classloading.ApplicationClassloader.getAnnotatedClasses(ApplicationClassloader.java:491)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:127)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:515)
	at play.Play.start(Play.java:537)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.ClassNotFoundException: snippet.Snippet
	at java.lang.ClassLoader.findClass(ClassLoader.java:358)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at play.classloading.ApplicationClassloader.loadClass(ApplicationClassloader.java:91)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:98)
	... 7 more
[2016-09-06 16:00:09,945] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ajpe25m
Internal Server Error (500) for request GET /buyer/tasks2

Oops: ClassNotFoundException
An unexpected error occured caused by exception ClassNotFoundException: snippet.Snippet

play.exceptions.UnexpectedException: Unexpected Error
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:100)
	at play.classloading.ApplicationClassloader.getAnnotatedClasses(ApplicationClassloader.java:491)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:127)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:515)
	at play.Play.start(Play.java:537)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.ClassNotFoundException: snippet.Snippet
	at java.lang.ClassLoader.findClass(ClassLoader.java:358)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at play.classloading.ApplicationClassloader.loadClass(ApplicationClassloader.java:91)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at play.classloading.ApplicationClasses.getAnnotatedClasses(ApplicationClasses.java:98)
	... 7 more
[2016-09-06 16:45:37,622] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [tidStr] Cannot be lower than 1
[2016-09-06 16:46:31,927] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-06 16:47:24,139] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [tidStr] Cannot be lower than 1
[2016-09-06 17:08:21,945] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@71akag53n
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/models/Task.java around line 414)
The file /app/models/Task.java could not be compiled. Error raised is : The method uploadAndSetSubwayItemPic(TaskMapper, boolean) is undefined for the type Task

play.exceptions.CompilationException: The method uploadAndSetSubwayItemPic(TaskMapper, boolean) is undefined for the type Task
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 17:08:22,664] ERROR [jobs-thread-7] play.Logger.niceThrowable(Logger.java:604) - 

@71akag53p
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/models/Task.java around line 414)
The file /app/models/Task.java could not be compiled. Error raised is : The method uploadAndSetSubwayItemPic(TaskMapper, boolean) is undefined for the type Task

play.exceptions.CompilationException: The method uploadAndSetSubwayItemPic(TaskMapper, boolean) is undefined for the type Task
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-06 17:19:13,105] ERROR [play-thread-1] models.Task2.save(Task2.java:525) - 
### Error committing transaction.  Cause: org.apache.ibatis.executor.BatchExecutorException: models.mappers.TaskMapper2.insert (batch index #1) failed. Cause: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
### Cause: org.apache.ibatis.executor.BatchExecutorException: models.mappers.TaskMapper2.insert (batch index #1) failed. Cause: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error committing transaction.  Cause: org.apache.ibatis.executor.BatchExecutorException: models.mappers.TaskMapper2.insert (batch index #1) failed. Cause: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
### Cause: org.apache.ibatis.executor.BatchExecutorException: models.mappers.TaskMapper2.insert (batch index #1) failed. Cause: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.commit(DefaultSqlSession.java:177)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.commit(DefaultSqlSession.java:169)
	at models.Task2.save(Task2.java:523)
	at controllers.TaskPublish.saveTask2(TaskPublish.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: org.apache.ibatis.executor.BatchExecutorException: models.mappers.TaskMapper2.insert (batch index #1) failed. Cause: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:127)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:114)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:109)
	at org.apache.ibatis.executor.BaseExecutor.commit(BaseExecutor.java:201)
	at org.apache.ibatis.executor.CachingExecutor.commit(CachingExecutor.java:104)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.commit(DefaultSqlSession.java:174)
	... 29 more
Caused by: java.sql.BatchUpdateException: Data truncation: Data too long for column 'item_subway_pic' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1819)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1281)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeBatch(NewProxyPreparedStatement.java:2465)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:85)
	at com.sun.proxy.$Proxy20.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:103)
	... 34 more
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Data too long for column 'item_subway_pic' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3833)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2145)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1777)
	... 43 more
